packages/scxa-tsne-plot/html/Demo.js

```diff
@@ -172,34 +174,48 @@ class Demo extends React.Component {
             <button className={`button`} type="submit">Submit</button>
           </form>
         </div>
+   */
 
+  render() {
+    return(
+       <div className={`row column expanded`}>
         <TsnePlotView
-          atlasUrl={`https://wwwdev.ebi.ac.uk/gxa/sc/`}
+          atlasUrl={`http://localhost:8080/gxa/sc/`}
```

Comment:
Please change the localhost URL to public or dev URL, please. So that someone running the demo in the future doesn't have to run anything on their local machine.